{"name": "test_enterDataInForm", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <pages.ContactUsFormPage.ContactForm object at 0x7fe78e827730>, text = 'Turkey'\nlocatorValue = 'Enter Address', locatorType = 'text'\n\n    def sendText(self, text, locatorValue, locatorType=\"id\"):\n        element = None\n        try:\n            locatorType = locatorType.lower()\n            element = self.getElement(locatorValue, locatorType)\n>           element.send_keys(text)\nE           AttributeError: 'NoneType' object has no attribute 'send_keys'\n\n../base/BasePage.py:74: AttributeError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.ContactUsFormTest.ContactFormTest testMethod=test_enterDataInForm>\n\n    @pytest.mark.run(order=2)\n    def test_enterDataInForm(self):\n        self.cf.enterName()\n        self.cf.enterEmail()\n>       self.cf.enterAddress()\n\nContactUsFormTest.py:20: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../pages/ContactUsFormPage.py:38: in enterAddress\n    self.sendText(\"Turkey\", self._enterAddress, \"text\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.ContactUsFormPage.ContactForm object at 0x7fe78e827730>, text = 'Turkey'\nlocatorValue = 'Enter Address', locatorType = 'text'\n\n    def sendText(self, text, locatorValue, locatorType=\"id\"):\n        element = None\n        try:\n            locatorType = locatorType.lower()\n            element = self.getElement(locatorValue, locatorType)\n            element.send_keys(text)\n            self.log.info(\n                \"Send text  on Element with LocatorType: \" + locatorType + \" and with the locatorValue :\" + locatorValue)\n        except:\n            self.log.info(\n                \"Unable to send text on Element with LocatorType: \" + locatorType + \" and with the locatorValue :\" + locatorValue)\n            self.takeScreenshot(locatorType)\n>           assert False\nE           AssertionError\n\n../base/BasePage.py:81: AssertionError"}, "steps": [{"name": "Entered Name", "status": "passed", "start": 1593918265785, "stop": 1593918265786}, {"name": "Entered Email", "status": "passed", "start": 1593918267146, "stop": 1593918267146}], "attachments": [{"name": "text", "source": "da5adb79-58cd-48fe-b3e0-7d2e07d315ae-attachment.png", "type": "image/png"}, {"name": "log", "source": "7a65ff4f-162f-4aeb-b0d9-88f189b78de1-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "30f877c8-0dc8-465e-888a-693871fcc9b5-attachment.txt", "type": "text/plain"}], "start": 1593918264844, "stop": 1593918293605, "uuid": "d8bc02f6-cd57-4f99-8f79-d5c7f74bcdf3", "historyId": "e52bdc27212ebfa017dda776df563de5", "testCaseId": "39a3f98e6d10b36122aedc8f3126c647", "fullName": "TestSuite.ContactFormTest#test_enterDataInForm", "labels": [{"name": "tag", "value": "run(order=2)"}, {"name": "tag", "value": "@pytest.mark.usefixtures('beforeClass', 'beforeMethod')"}, {"name": "suite", "value": "TestSuite"}, {"name": "subSuite", "value": "ContactFormTest"}, {"name": "host", "value": "Bobs-MBP.fios-router.home"}, {"name": "thread", "value": "43061-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestSuite"}]}